cmd/cstor-pool-mgmt/controller/replica-controller/handler.go

```diff
@@ -81,7 +81,7 @@ func (c *CStorVolumeReplicaController) cVREventHandler(operation common.QueueOpe
 		return string(apis.CVRStatusOffline), fmt.Errorf("Pool not present")
 	}
 
-	fullVolName := string(pool.PoolPrefix) + cVR.Labels["cstorpool.openebs.io/uid"] + "/" + cVR.Labels["cstorvolume.openebs.io/uid"]
+	fullVolName := string(pool.PoolPrefix) + cVR.Labels["cstorpool.openebs.io/uid"] + "/" + cVR.Labels["cstorvolume.openebs.io/name"]
```

Comment:
can some comments explain why we are doing this?
can we avoid use of `/`. Historically, it has proved erroneous.
why do we need to use uid since uid is not human readable.

Reply:
yes sure, I'll add the comments. zfs requires command to be poolname + "/" + volname.

Reply:
For cvr, I have addressed the change from uid to name. For cstorpool, it is slightly different. we are making use of both uid and name of cr. There is no additional uid, but actual cstorpool CR uid, we are using